packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -69,6 +75,45 @@ export const logDislikeGeneratedAnswer = (): CustomAction =>
     }
   );
 
+export const logGenerativeQuestionFeedback = (
+  feedback: GeneratedAnswerFeedback
+): CustomAction =>
+  makeAnalyticsAction(
+    'analytics/generatedAnswer/sendFeedback',
+    AnalyticsType.Custom,
+    (client, state) => {
+      const generativeQuestionAnsweringId =
+        generativeQuestionAnsweringIdSelector(state);
+      if (!generativeQuestionAnsweringId) {
+        return null;
+      }
```

Comment:
Added this check cause I found that we are adding it in previously made actions, but honestly I can't imagine a situation where this would happen, but I see it as safety net  to avoid sending analytics without this valuable information.